packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/pch.h

```diff
@@ -3,6 +3,8 @@
 
 #pragma once
 
+#define NOMINMAX
```

Comment:
>NOMINMAX [](start = 8, length = 8)

I must add the NOMINMAX because I have stated to use std::numeric_limits<T>::max and these Windows.h macros fail the compilation.